js/serial.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 var serial = {
+    connectionType:  'serial', // 'serial' or 'tcp'
```

Comment:
Where it is now is good. Having new stuff at the beginning is confusing. And don't worry about English, it isn't my first language either, and the same goes for many contributors here.